api/.eslintrc

```diff
@@ -0,0 +1,32 @@
+{
+  "env": {
+    "browser": false,
+    "jquery": false
+  },
+  "rules": {
+    "eqeqeq": "off",
```

Comment:
I see we also did this in JSHint. If we leave this on what breaks? If we can get away with leaving it on we should.

Reply:
Something definitely broke, I remember, that's why I added it